src/plugins/management/public/legacy/sections_register.js

```diff
@@ -52,6 +52,14 @@ export class LegacyManagementAdapter {
       icon: 'logoKibana',
     });
 
+    this.main.register('ingest', {
```

Comment:
You shouldn't have to add this here.
You can instead add the `management` plugin as a dependency to your plugin.
Then use the `management.sections.register` API.
See `x-pack/plugins/security/public/management/management_service.ts` for reference.

Reply:
I think this is left over from a previous iteration and we can remove it.

At one point the UI was under the Management section but we've since moved it out to the same level as the other plugins.

@hbharding, let me know if that's incorrect or missing something

Reply:
yep, we can remove this

Reply:
removed in dcade4c